buildsystem/python.cmake

```diff
@@ -169,7 +183,6 @@ function(add_cython_modules)
 				set(STANDALONE_NEXT FALSE)
 			else()
 				set_target_properties("${TARGETNAME}" PROPERTIES LINK_DEPENDS_NO_SHARED 1)
-				target_link_libraries("${TARGETNAME}" "${PYEXT_LINK_LIBRARY}")
```

Comment:
then set some other variable in the line where embed_next is still active, which then triggers the `dynamic_lookup` addition here. I just noticed that the `set_target_properties` might override already existing flags, we might want to prevent this.